### PR TITLE
added failing ChangeIfElseValueAssignToEarlyReturnRector test, regarding lost comments

### PR DESCRIPTION
**Pull request by clxmstaab**
demo https://getrector.org/demo/688523bc-8f43-4f00-8401-04dee41b083b#result

### CODE REVIEW

File: rules/solid/tests/Rector/If_/ChangeIfElseValueAssignToEarlyReturnRector/Fixture/lost_comment.php.inc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<?php
+
+namespace Rector\SOLID\Tests\Rector\If_\ChangeIfElseValueAssignToEarlyReturnRector\Fixture;
+
+class LostComment
+{
+    protected function replaceUrl($internalUrl, $publicUrl, $contenttext)
+    {
+        if (null === $publicUrl) {
+            // parameters could not be resolved into a url
```
Comment (clxmstaab) [inline rules/solid/tests/Rector/If_/ChangeIfElseValueAssignToEarlyReturnRector/Fixture/lost_comment.php.inc:31]: this line is currently lost, when processing

File: rules/solid/tests/Rector/If_/ChangeIfElseValueAssignToEarlyReturnRector/Fixture/lost_comment.php.inc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            return preg_replace('@href=(["\'\s]*'.preg_quote($internalUrl, '@').'["\'\s]*)@', '', $contenttext);
+        }
+
+        // else comment
```
Comment (clxmstaab) [inline rules/solid/tests/Rector/If_/ChangeIfElseValueAssignToEarlyReturnRector/Fixture/lost_comment.php.inc:36]: just added another case of comment

**Comment by clxmstaab** [comment]
fixed the bug. tests are green now.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**